.gitignore

```diff
@@ -13,6 +13,7 @@ packages/*/test/**/*.d.ts
 
 package-lock.json
 *.log
+**/.*.swp
```

Comment:
A comprehensive ignore pattern would actually be this:
```
[._]*.s[a-w][a-z]                                                                                                                                                                                                                                                              
[._]s[a-w][a-z]
```

Reply:
Do you have some examples of other swap-type files which these patterns would match?

Also: RIP PR, we hardly knew ye.

Reply:
If vim sees that a swap file exists and you tell it to edit anyway, it will create a `.swo` file, and it goes up the alphabet from `p` if even that is present.